hbp_knowledge/ms/cappa2017.bel

```diff
@@ -0,0 +1,345 @@
+SET Gender = {"Female", "Male"}
+path(MESH:"Body Mass Index") -> path(ORPHANET:477738)
+UNSET Gender
+
+####pathogenic TH17?
+SET Evidence = "Though human studies are confounded due to reliance on patient recollection and nonspecific survey-based questionnaires, high salt diet has been shown in EAE models to increase risk of autoimmune disease and drive severity of EAE due to induction of pathogenic TH17 cells (52). One"
+SET MeSHDisease = "Encephalomyelitis, Autoimmune, Experimental"
+bp(MESH:"Diet, Sodium-Restricted") pos path(EFO:"autoimmune disease")
+UNSET MeSHDisease
+bp(MESH:"Diet, Sodium-Restricted") -| p(MESH:"Th17 Cells", var("?"))
+p(MESH:"Th17 Cells", var("?")) -> path(EFO:"experimental autoimmune encephalomyelitis")
+bp(MESH:"Diet, Sodium-Restricted") -| path(EFO:"experimental autoimmune encephalomyelitis")
+
+SET Evidence = "The bacterial milieu within the gastrointestinal tract has a substantial influence on an individual’s immune profile and response – including development of the immune system, protection against pathogens, and maintenance of immune response (57)."
+path(MESH:"Gastrointestinal Microbiome") reg a(MESH:"Immune System")
+path(MESH:"Gastrointestinal Microbiome") reg path(MESH:Immunity)
+path(MESH:"Gastrointestinal Microbiome") reg bp(GO:"immune response")
+
+SET Evidence = "Indeed, pediatric MS case-control studies have demonstrated differences in gut flora (at a taxa-level) between cohorts(61)."
+path(ORPHANET:477738) reg path(MESH:"Gastrointestinal Microbiome")
+
+SET Evidence = "MS disease-modifying therapies and vitamin D supplementation have a modulatory effect on the human
+gut microbial composition (62)."
+a(MESH:"Vitamin D") reg path(MESH:"Gastrointestinal Microbiome")
+
+SET Evidence = "To-date, taking into account the entire human genome, the MHC, located on chromosome 6, comprises the single largest genetic contribution to MS susceptibility. HLA genes encode for cell surface glycoproteins with a primary role in presenting antigenic peptide fragments to T-cells (74)"
+p(HGNCGENEFAMILY:"Histocompatibility complex") -> path(DOID:"multiple sclerosis")
+g(HGNCGENEFAMILY:"Histocompatibility complex") -> p(MESH:"Membrane Glycoproteins")
+p(MESH:"Membrane Glycoproteins") -> bp(GO:"T cell antigen processing and presentation")
+
+SET Evidence = "Deeper investigation revealed these to be part of an HLA haplotype – in particular, the HLA-DRB1*15:01 allele has consistently been shown to be an independent risk factor for MS, particularly in populations of European ancestry, with an odds ratio (OR) of approximately 3 (75)."
+g(IMGTHLA:HLA00865) pos path(DOID:"multiple sclerosis")
+
+SET Evidence = "the HLA-DRB1*15:03 allele is a risk factor in African Americans, while the HLA-A*02 allele appears to confer a protective effect against MS, with an OR of approximately 0.6 (79)."
+p(IMGTHLA:HLA00870) -> path(DOID:"multiple sclerosis")
+path(NCIT:C69408) -| path(DOID:"multiple sclerosis")
+
```

Comment:
we need to discuss and agree on the coding of HLAs in the curation team meeting; 
path(NCIT:C69408) can not be used to code for HLA-A*02; the allele is not a pathology nor is the NCIT term "HLA-A2 Positive Cells Present" which described a laboratory test result, a suitable way to code the allele

Reply:
@YojanaGadiya Did we decide on how to handle that? I thought we discussed that issue?